source/ui/PGM/PGMBeamStatusView.cpp

```diff
@@ -0,0 +1,61 @@
+#include "PGMBeamStatusView.h"
+#include "beamline/PGM/PGMBeamline.h"
+
+#include <QGridLayout>
+#include <QLabel>
+#include <QGroupBox>
+
+
+PGMBeamStatusView::PGMBeamStatusView(QWidget *parent) :
+    QWidget(parent)
+{
+    ringCurrentLabel_ = new QLabel("Ring Current: ");
```

Comment:
@iainworkman I would like to do more code refactor in #2355 to make this ring current display stuff universal to all the beamlines.

If you are not against it, we will live with this changes and refactor it later.